monitoring/monitoring-overview.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="monitoring-overview"]
+= Monitoring overview
+include::modules/common-attributes.adoc[]
+:context: monitoring-overview
+
+toc::[]
+
+[id="about-openshift-container-platform-monitoring"]
+== About {product-title} Container Platform monitoring
```

Comment:
This title renders as "About OpenShift Container Platform Container Platform monitoring". You can remove "Container Platform" from the title, because `{product-title}` resolves to "OpenShift Container Platform" when built as part of the OCP distribution. This will also prevent other distributions from having "Container Platform" appended to the product name if this title is used.

Reply:
oops my mistake Thanks @pneedle-rh